README.md

```diff
@@ -48,6 +48,27 @@ script/server --mock-api
 
 API endpoint stubbing is set up in `/wiremock/scripts/stubApis.ts`.
 
+
+### Local user accounts
+
+There are three user accounts with different roles that can be used when running the application locally:
+
+#### POM User with roles: `POM`, `ACP_REFERRER`
+
+- Username: `ACP_POM_USER`
+- Password: `password123456`
+
+#### Programme Team user with roles: `ACP_PROGRAMME_TEAM`,  `ACP_REFERRER`
+
+- Username: `ACP_PT_REFERRER_USER`
+- Password: `password123456`
+
+#### Programme Team user with roles: `ACP_PROGRAMME_TEAM`
+
+- Username: `ACP_PT_USER`
+- Password: `password123456`
```

Comment:
Would this be better in a table?

| Type | Roles | Username | Password |
| ---- | ---- | ---- | ---- |
| POM | `POM`, `ACP_REFERRER` | `ACP_POM_USER` | `password123456` |
| Programme team | `ACP_PROGRAMME_TEAM`,  `ACP_PT_REFERRER_USER ` | `ACP_POM_USER` | `password123456` |
| Programme team | `ACP_PROGRAMME_TEAM` | `ACP_PT_USER ` | `password123456` |

Reply:
Or:

| Type | POM role | ACP_REFERRER role | ACP_PROGRAMME_TEAM role | Username |
| ---- | ---- | ---- | ---- | ---- |
| POM | x | x | | `ACP_POM_USER` |
| Programme team | | x | x | `ACP_POM_USER`|
| Programme team | | | x | `ACP_PT_USER ` |

All passwords are `password123456`

Reply:
Good shout, have combined this and your suggestion above into one!